src/moscot/backends/ott/output.py

```diff
@@ -100,11 +98,29 @@ def plot_errors(
 
         Parameters
         ----------
-        %(plot_costs.parameters)s
+        last
+            Whether to plot only the ``last`` errors. If :obj:`None`, plot all errors.
```

Comment:
shouldn't be this be like the above [line 55]?
"last
            How many of the last steps of the algorithm to plot. If :obj:`None`, plot the full curve."